internal/cmd/postgresflex/user/user.go

```diff
@@ -0,0 +1,35 @@
+package user
+
+import (
+	"github.com/stackitcloud/stackit-cli/internal/cmd/postgresflex/user/create"
+	"github.com/stackitcloud/stackit-cli/internal/cmd/postgresflex/user/delete"
+	"github.com/stackitcloud/stackit-cli/internal/cmd/postgresflex/user/describe"
+	"github.com/stackitcloud/stackit-cli/internal/cmd/postgresflex/user/list"
+	resetpassword "github.com/stackitcloud/stackit-cli/internal/cmd/postgresflex/user/reset-password"
+	"github.com/stackitcloud/stackit-cli/internal/cmd/postgresflex/user/update"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/utils"
+
+	"github.com/spf13/cobra"
+)
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "user",
+		Short: "Provides functionality for PostgreSQL Flex users",
+		Long:  "Provides functionality for PostgreSQL Flex users",
```

Comment:
```suggestion
		Long:  "Provides functionality for PostgreSQL Flex users.",
```

Reply:
Applied other changes to descriptions as well